crates/admin/src/cluster_controller/service.rs

```diff
@@ -160,6 +165,18 @@ impl ClusterControllerHandle {
         rx.await.map_err(|_| ShutdownError)
     }
 
+    pub async fn get_log_chain(&self, log_id: LogId) -> Result<Option<Arc<Chain>>, ShutdownError> {
```

Comment:
Right now this is redundant as the same information can be obtained via `GetClusterState` - however I wanted to see how to implement a new command, as I think we might need to fetch more. For example, additional log chain details not currently present in `ClusterState` - I was hoping to show the latest persisted LSN for the tail segment for one.